app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -262,17 +275,35 @@ class LuciBuildService {
       }
 
       final Build scheduleBuild = response.scheduleBuild;
+      if (scheduleBuild.tags == null) {
```

Comment:
what does `tags==null` mean?

Reply:
There are two types of errors:

* An error that has tags information that we can use to update the check as a failed task
* An error with no tags that we can not provide any feedback, tags == null is this case.